platform-tests/src/test/java/org/junit/platform/launcher/listener/CustomTestExecutionListener.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.platform.launcher.listener;
+
+import org.junit.platform.engine.TestExecutionResult;
+import org.junit.platform.engine.reporting.ReportEntry;
+import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestIdentifier;
+import org.junit.platform.launcher.TestPlan;
+
+public class CustomTestExecutionListener implements TestExecutionListener {
+
+	public int testPlanExecutionStarted;
```

Comment:
Please convert each `int` to an `AtomicInteger` to ensure that concurrent execution of listeners is supported from the start.